www/front_src/src/Resources/Graph/Performance/TimePeriods/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { map, pick } from 'ramda';
+
+import { Paper, makeStyles, ButtonGroup, Button } from '@material-ui/core';
+
+import { TimePeriodId, timePeriods } from '../../../Details/tabs/Graph/models';
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    padding: theme.spacing(2),
+  },
+}));
+
+interface Props {
+  selectedTimePeriodId: string;
+  onChange: (timePeriod: TimePeriodId) => void;
+  disabled?: boolean;
+}
+
+const timePeriodOptions = map(pick(['id', 'name']), timePeriods);
+
+const TimePeriods = ({
```

Comment:
```suggestion
const TimePeriodButtonGroup = ({
```

So that it’s clearer